src/static-resources/.well-known/security.txt

```diff
@@ -0,0 +1,7 @@
+Contact: https://www.nijmegen.nl/diensten/privacy/beveiligings-of-datalek-melden/
```

Comment:
Goed dat ie er is, qua beheerlast zou ik dit willen vervangen door een redirect naar de centrale security.txt (wederom, kan als nieuw issue, geen blocker)

Reply:
daar was ik al mee aan het pielen geweest, maar kreeg ik nog niet werkend.